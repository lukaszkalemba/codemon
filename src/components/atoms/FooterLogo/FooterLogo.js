import React from 'react';
import Icon from 'components/atoms/Icon';
import logo from 'assets/svgs/logo_codemon-white.svg';
import S from './FooterLogo.styles';

const FooterLogo = () => (
  <S.Wrapper>
    <Icon src={logo} />
  </S.Wrapper>
);

export default FooterLogo;
