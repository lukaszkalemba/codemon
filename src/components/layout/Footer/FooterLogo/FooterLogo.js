import React from 'react';

const FooterLogo = () => <div>footer logo</div>;

export default FooterLogo;
