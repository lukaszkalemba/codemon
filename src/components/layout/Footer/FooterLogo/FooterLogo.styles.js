import styled from 'styled-components';

const Wrapper = styled.div`
  display: none;
  position: relative;
  z-index: ${({ theme }) => theme.zindex.positive1};

  ${({ theme }) => theme.media.laptop`
    display: flex;
    justify-content: center;
    align-items: center;
    margin: 0 auto;
    width: 100%;
    height: 100%;

    svg {
      width: 100%;
      height: 100%;
    }
  `}

  ${({ theme }) => theme.media.largeDesktop`
    width: 65%;
  `}
`;

export default { Wrapper };
