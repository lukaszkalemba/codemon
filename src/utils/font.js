export default {
  size: {
    xxxs: '1rem',
    xxs: '1.3rem',
    xs: '1.6rem',
    s: '2.2rem',
    m: '3rem',
    l: '4rem',
    xl: '5.4rem',
    xxl: '8.5rem',
    xxxl: '10rem',
  },

  weight: {
    regular: '400',
    medium: '500',
    bold: '700',
  },

  family: {
    montserrat: 'Montserrat',
  },
};
