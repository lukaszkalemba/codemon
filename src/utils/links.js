export default [
  {
    id: 0,
    path: '/projekty',
    name: 'projekty',
  },
  {
    id: 1,
    path: '/blog',
    name: 'blog',
  },
  {
    id: 2,
    path: '/wycena',
    name: 'wycena',
  },
];
