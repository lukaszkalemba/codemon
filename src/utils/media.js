export default {
  phone: '576px',
  tablet: '768px',
  desktop: '1150px',
  largeDesktop: '1600px',
};
