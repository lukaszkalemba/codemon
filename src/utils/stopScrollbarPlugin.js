import { ScrollbarPlugin } from 'smooth-scrollbar';

class stopScrollbarPlugin extends ScrollbarPlugin {
  static pluginName = 'stopScrollbar';

  static defaultOptions = {
    open: false,
  };

  transformDelta(delta) {
    return this.options.open ? { x: 0, y: 0 } : delta;
  }
}

export default stopScrollbarPlugin;
