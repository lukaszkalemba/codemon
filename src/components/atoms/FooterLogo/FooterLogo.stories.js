import React from 'react';
import FooterLogo from './FooterLogo';

export const footerLogo = () => <FooterLogo />;

export default {
  component: FooterLogo,
  title: 'Atoms|FooterLogo',
};
