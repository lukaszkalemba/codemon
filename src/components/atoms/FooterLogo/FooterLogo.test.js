import React from 'react';
import { render } from 'utils/test-utils';
import FooterLogo from './FooterLogo';

describe('<FooterLogo />', () => {
  it('matches snapshot', () => {
    const { container } = render(<FooterLogo />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
