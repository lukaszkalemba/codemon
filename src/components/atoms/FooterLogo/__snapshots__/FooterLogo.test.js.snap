// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterLogo /> matches snapshot 1`] = `
<div
  class="sc-gtssRu iKsAlc"
  data-testid="footer-logo"
>
  <div
    class="sc-bdnylx levWdn"
  >
    <svg
      fill="none"
      height="16"
      viewBox="0 0 65 16"
      width="65"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 1H61L42.5 14.5"
        stroke="white"
        stroke-width="2"
      />
    </svg>
  </div>
</div>
`;
